python/lsst/sims/catUtils/utils/ObservationMetaDataGenerator.py

```diff
@@ -221,6 +222,8 @@ def getObservationMetaData(self, obsHistID=None, expDate=None, fieldRA=None, fie
 
                 nConstraints += 1
 
+        query += ' GROUP BY expMJD'
```

Comment:
Yeah.  I have yet to see things come back out of chronological order, but I haven't verified that that is implied by GROUP BY